waspc/src/Wasp/Generator/WebAppGenerator.hs

```diff
@@ -320,10 +320,13 @@ genViteConfig spec = return $ C.mkTmplFdWithData tmplFile tmplData
           "defaultClientPort" .= C.defaultClientPort
         ]
 
-    makeCustomViteConfigJsImport :: Path' (Rel SourceExternalCodeDir) File' -> JsImport
+    makeCustomViteConfigJsImport :: Path' (Rel WaspProjectDir) File' -> JsImport
```

Comment:
This function creates an import for the custom config, it does it relatively from the `web-app` to the user project dir.